elements/pfe-navigation/src/_variables-mixins.scss

```diff
@@ -6,11 +6,39 @@ $LOCAL: navigation;
 /// PFE-Local setup variables
 /// These variables will be namespaced and become css vars
 $LOCAL-VARIABLES: (
```

Comment:
@castastrophe 
The main event.
After we meet my next step will be to 'default' certain local vars to theme vars.
Also think there'll end up being more vars, be curious on your impression of what could use vars